docker-compose.yml

```diff
@@ -370,9 +385,22 @@ services:
       - "8126"
     ports:
       - "127.0.0.1:${DD_METRIC_AGENT_PORT}:8125/udp"
-      - "127.0.0.1:${DD_TRACE_AGENT_PORT}:8126"
+      - "127.0.0.1:8126:8126"
     volumes:
       - ddagent_var_run:/var/run/datadog
+
+  testagent:
+    image: ghcr.io/datadog/dd-apm-test-agent/ddapm-test-agent:latest
+    ports:
+        - "127.0.0.1:9126:9126"
+    environment:
+        - LOG_LEVEL=DEBUG
+        - TRACE_LANGUAGE=ruby
+        - DD_TRACE_AGENT_URL=http://ddagent:8126
```

Comment:
Because we reference `ddagent` here, I believe we are missing a
```yaml
depends_on:
  - ddagent
```
for the `testagent` docker service.